cmd/pebble/cmd_replan_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"fmt"
+	pebble "github.com/canonical/pebble/cmd/pebble"
+	"gopkg.in/check.v1"
+	"net/http"
+)
+
+func (s *PebbleSuite) TestReplan(c *check.C) {
+	failGetChange := false
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		fakeChange := `{
+  "id":   "43",
+  "kind": "replan",
+  "summary": "...",
+  "status": "Done",
+  "ready": true,
+  "spawn-time": "2016-04-21T01:02:03Z",
+  "ready-time": "2016-04-21T01:02:04Z",
+  "tasks": []
+}`
+		if r.URL.Path == "/v1/changes" {
```

Comment:
It's unclear to me why `/v1/changes` is needed (the test passes fine without it, so I don't think it is).

Reply:
Now that you speak of it, I don't remember what was my reasoning behind that. You are absolutely correct, thanks for your feedback!